AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -79,6 +79,11 @@
     '2000Nits': 2,
     '4000Nits': 3,
 }
+# Control Type options for the Exposure Control component.
+CONTROL_TYPE = {
```

Comment:
can we rename this to `EXPOSURE_CONTROL_TYPE` to make it more specific and less generalized?

Reply:
Done